website/packages/docs/src/pages/how-it-works.mdx

```diff
@@ -0,0 +1,163 @@
+---
+order: 3
+section: 1-Getting started
+---
+
+# How it Works
+
+Once you've [installed and configured](/installation) everything, you can start using it,
+but it's best to explain what happens under the hood.
+
+Compiled uses Babel (with pre-configuration options via Webpack and Parcel) to transform styles
+into atomic CSS determined at build time, resulting in resulting in **very performant components**.
```

Comment:
Typo?

Reply:
I swear I do this a lot, I don't understand how…
```suggestion
into atomic CSS determined at build time, resulting in **very performant components**.
```